examples/secure_token_storage.py

```diff
@@ -2,13 +2,15 @@
 # You should load the token as an environment variable, then access it like this:
 
 import os
+from nextcord.ext import commands
 
 # There are other ways to load environment variables but this is one
 from dotenv import load_dotenv # https://pypi.org/project/python-dotenv/
 
 # load_dotenv reads from a file called .env in the same directory as the python files which should roughly look like TOKEN="1234567890"
```

Comment:
Oh yeah one thing, that comment is incorrect "should look like TOKEN=" but the env taken is BOT_TOKEN